zio/zngio/sort.go

```diff
@@ -0,0 +1,170 @@
+package zngio
```

Comment:
Like we discussed in slack, this sort implementation will get removed, and would need to be in a separate PR where it would be considered to replace the existing logic in the sort proc.